### PR TITLE
Update the following RL notebooks:

### PR DESCRIPTION
**Pull request by goelakash**
*Issue #, if available:*

*Description of changes:*
- rl_cartpole_ray_gymEnv.ipynb
- rl_network_compression_ray_custom.ipynb
- rl_gamerserver_ray.ipynb
- rl_roboschool_ray_automatic_model_tuning.ipynb
- rl_roboschool_stable_baselines.ipynb

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
```
Comment (yijiezh) [inline reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb:27]: <p>supposed to change the param in this cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (goelakash) [reply reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb:27]: <p>Updated</p>

Comment (yijiezh) [inline reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb:27]: <p>I guess TensorFlowModel() require to specify version number?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1738/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (goelakash) [reply reinforcement_learning/rl_game_server_autopilot/sagemaker/rl_gamerserver_ray.ipynb:27]: <p>Added</p>

**Review by yijiezh — APPROVED** [review]
LGTM

**Pull request merged**